hpaction/build_hpactionvars.py

```diff
@@ -64,10 +66,103 @@ def create_complaint(area: str, description: str) -> hp.TenantComplaints:
     )
 
 
+def nycdb_addr_to_hp_state(address: nycdb.models.Address) -> hp.LandlordAddressStateMC:
+    # This is kind of yucky, because NYCDB/HPD provides state information as
+    # all-caps two-letter strings (e.g. "NY") while the HotDocs endpoint wants
+    # them fully spelled out (e.g. "New York"). However, in practice it doesn't seem
+    # to mind if we give it the two-letter strings, so we'll just do that.
```

Comment:
Um, I say "in practice it doesn't seem to mind" but I haven't actually verified this, I'm just guessing it's the case.  I should verify it before merging.

Reply:
Okay, I just verified it... um... it actually seems like the more granular address info might not be used _at all_, i.e. LHI is just asking for it so it knows that the full address it pastes together is valid, and it doesn't need to do that with our answers because we paste stuff together on our end.  I might be wrong though.